notebooks/official/model_evaluation/custom_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<blockquote>Locally, create an application using for serving using&nbsp;<code><a href="https://fastapi.tiangolo.com/tutorial/first-steps/" target="_blank">FastAPI</a></code>&nbsp;Python package.</blockquote><p>using for serving using? I think this needs to be edited :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>removed the extra "using".</p>